_posts/0000-01-02-btkh95.md

```diff
@@ -0,0 +1 @@
+/
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!